.github/workflows/rust-tests.yml

```diff
@@ -84,9 +84,9 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
     - name: Install Rust
-      uses: dtolnay/rust-toolchain@stable
+      uses: actions-rs/toolchain@v1
```

Comment:
Please don't use this one. `actions-rs` is abandonned. `dtolnay` is the currently maintained one. You can just set `env.RUST_TOOLCHAIN` to `nightly` instead.